src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4861blockcommentstyle/InputFormattedBlockCommentStyle.java

```diff
@@ -0,0 +1,910 @@
+    String s = new String("A" + "B" + "C");
+    // comment
+  }
+
+  /** some javadoc. */
+  public void foo32() {
+
+    String s = new String("A" + "B" + "C");
+    // odd indentation comment
+  }
+
+  /** some javadoc. */
+  public void foo33() {
+    // comment
+
+    this.foo22();
+    // odd indentation comment
+  }
+
+  /** some javadoc. */
+  public void foo34() throws Exception {
+    throw new Exception("", new Exception());
+    // comment
+  }
+
+  /** some javadoc. */
+  public void foo35() throws Exception {
+
+    throw new Exception("", new Exception());
+    // odd indentation comment
+  }
+
+  /** some javadoc. */
+  public void foo36() throws Exception {
+
+    throw new Exception("", new Exception());
+    // odd indentation comment
+  }
+
+  public void foo37() throws Exception {
+    throw new Exception("", new Exception());
+    // comment
+  }
+
+  /** some javadoc. */
+  public void foo38() throws Exception {
+
+    throw new Exception("", new Exception());
+    // odd indentation comment
+  }
+
+  /** some javadoc. */
+  public void foo39() throws Exception {
+
+    throw new Exception("", new Exception());
+    // odd indentation comment
+  }
+
+  /** some javadoc. */
+  public void foo40() throws Exception {
+    int a = 88;
+
+    throw new Exception("", new Exception());
+    // odd indentation comment
+  }
+
+  /** some javadoc. */
+  public void foo41() throws Exception {
+    int a = 88;
+    throw new Exception("", new Exception());
+    // comment
+  }
+
+  /** some javadoc. */
+  public void foo42() {
+    int a = 5;
+    if (a == 5) {
+      int b;
+      // comment
+    } else if (a == 6) {
+      /* foo */
+    }
+  }
+
+  /** some javadoc. */
+  public void foo43() {
+    try {
+      int a;
+      // comment
+    } catch (Exception e) {
+      /* foo */
+    }
+  }
+
+  /** some javadoc. */
+  public void foo44() {
+    int ar = 5;
+    // comment
+    ar = 6;
+    // comment
+  }
+
+  /** some javadoc. */
+  public void foo45() {
+    int ar = 5;
+    // comment
+
+    ar = 6;
+    // odd indentation comment
+  }
+
+  /** some javadoc. */
+  public void foo46() {
+
+    // comment
+    // block
+    // odd indentation comment
+  }
+
+  /** some javadoc. */
+  public void foo47() {
+    int a = 5;
+    // comment
+    // block
+    // comment
+  }
+
+  /** some javadoc. */
+  public void foo48() {
+
+    int a = 5;
+    // comment
+    // block
+    // odd indentation comment
+  }
+
+  /** some javadoc. */
+  public void foo49() {
+    // comment
+    // block
+    // ok
+  }
+
+  /** some javadoc. */
+  public void foo50() {
+    return;
+
+    // No NPE here!
+  }
+
+  /** some javadoc. */
+  public String foo51() {
+
+    return String.valueOf("11");
+    // odd indentation comment
+  }
+
+  public String foo52() {
+    return String.valueOf("11");
+    // comment
+  }
+}
+
+// violation below 'Top-level class ExtraClass1 has to reside in its own source file.'
+class ExtraClass1 {
```

Comment:
there were total 4 input files for this section, I created extra classes for every input files so had to add violation